scss/index.scss

```diff
@@ -0,0 +1,126 @@
+@import "imports/variables";
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+body {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Cantarell', sans-serif;
+}
+
+#scroll {
+  width: 1000px;
+  background-color: $mine-shaft-grey;
+  height: 700px;
+  margin: 0 auto;
+  overflow: auto;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+h2 {
+  margin-bottom: 0;
+  margin-top: 2%;
+}
+
+.poke-container {
+  width: 100%;
+  height: 150px;
+  outline-style: solid;
+  outline-offset: -2px;
+}
+
+.info {
+  width: 65%;
+  height: 100%;
+  float: left;
+  background-color: $poke-grey;
+  padding-left: 5%;
+}
+
+.upper {
+  height: 30%;
+}
+.poke-title {
+  width: 70%;
+  float: left;
+}
+
+.poke-id {
+  width: 30%;
+  float: left;
+}
+
+.lower {
+  height: 70%;
+}
+
+.type-holder {
+  width: 20%;
+  float: left;
+  height: 100%;
+  font-size: 1.2rem;
+}
+
+.poke-attributes {
+  width: 80%;
+  height: 100%;
+  float: left;
+  text-align: center;
+  overflow: auto;
+}
+
+.icon-container {
+  float: left;
+  height: 40%;
+  padding: 3% 12% 0 12%;
```

Comment:
Could this be achieved with a margin: auto?

Reply:
no